config/tests/pom.xml

```diff
@@ -32,6 +32,13 @@
         <!-- Rules Dependencies -->
         <dependency>
             <groupId>org.jboss.windup.rules.apps</groupId>
+            <artifactId>rules-base</artifactId>
```

Comment:
I believe that this can also be removed once the dependency lines are removed from the unit test.